scripts/qa_crowbarsetup.sh

```diff
@@ -1356,6 +1362,10 @@ EOF
         fi
     fi
 
+    if [[ $want_hcp = 1 ]]; then
+        add_hcp_repo
+    fi
```

Comment:
could be `[[ $want_hcp = 1 ]] && add_hcp_repo`
or do we expect the special case to grow?